src/data/community/channels.yml

```diff
@@ -10,10 +10,6 @@
   url: "https://slack.decred.org"
   img_src: "dcr-community-platforms-slack.png"
   name: Slack
--
-  url: "https://webchat.freenode.net/?channels=decred&uio=d4"
-  img_src: "dcr-community-platforms-irc.png"
```

Comment:
We should be able to remove `dcr-community-platforms-irc.png` as well as any other IRC related images.

Some of them are still referenced in `.css` files, so they may need a review also.

Reply:
Ah right, so the idea is to kill it altogether - I'll do a proper cleanup then.

Reply:
I believe that is correct yeah - @jzbz could you confirm?

Reply:
Correct. IRC gets nuked.

Reply:
Removed all references

Reply:
Link is still appearing in the nav menu (although the icon has been deleted)

Reply:
Thanks for spotting, it's gone now